packages/main/src/MultiComboBox.js

```diff
@@ -122,6 +122,18 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines whether the <code>ui5-input</code> is required.
```

Comment:
Sorry for not having noted that in my previous review, but this doc is incorrect. Should be multi cb instead of input here.